packages/suite/src/views/dashboard/components/AssetsView/components/AssetCard.tsx

```diff
@@ -11,55 +11,44 @@ import {
 } from 'src/components/suite';
 import { isTestnet } from '@suite-common/wallet-utils';
 import { CoinmarketBuyButton } from 'src/views/dashboard/components/CoinmarketBuyButton';
-import { borders, spacingsPx, typography } from '@trezor/theme';
+import { spacingsPx, typography } from '@trezor/theme';
 
-import { H2, Icon, SkeletonRectangle, variables } from '@trezor/components';
+import { Card, H2, Icon, SkeletonRectangle, variables } from '@trezor/components';
 import { useDispatch } from 'react-redux';
 import { useAccountSearch, useLoadingSkeleton } from 'src/hooks/suite';
 import { goto } from 'src/actions/suite/routerActions';
 import { ArrowIcon, AssetInfo, AssetInfoSkeleton } from './AssetInfo';
 import { AssetFiatBalance } from '@suite-common/assets';
 import { FiatHeader } from '../../FiatHeader';
 
-const BuyContainer = styled.div`
+const BuyContainerCard = styled(Card)`
```

Comment:
I think it shouldn't be a `Card`. Just follow elevation background, but margins/paddings/border-radiuses are different.

Reply:
fixed on call